hamlet/settings/local.py

```diff
@@ -4,7 +4,7 @@
 
 from .heroku import *  # noqa
 
-ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1:8000']
+ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', 'localhost']
```

Comment:
This actually needs to be `127.0.0.1:8000` to work with `python manage.py runserver`.

Reply:
(If you need `127.0.0.1` for Docker purposes, please put both into `ALLOWED_HOSTS`.)

Reply:
I was going off the Django docs that explicitly say not to include the port here: https://docs.djangoproject.com/en/2.0/ref/settings/#allowed-hosts. From my understanding, having the port means it will never match. If I'm wrong let me know.